addon/components/ember-popper-base.js

```diff
@@ -161,7 +184,9 @@ export default class EmberPopperBase extends Component {
       || isPopperTargetDifferent
       || eventsEnabled !== this._eventsEnabled
       || modifiers !== this._modifiers
-      || placement !== this._placement;
+      || placement !== this._placement
+      || onCreate !== this._onCreate
+      || onUpdate !== this.onUpdate;
```

Comment:
`this.onUpdate` => `this._onUpdate` (missing underscore)

Reply:
Oups, sorry about that. Just fixed this...